src/CreditCardForm.js

```diff
@@ -70,6 +70,9 @@ export default class CreditCardForm extends Component {
     linkStyling: PropTypes.object,
 
     /** How wide you want the content area of `<PaymentMethod />`. */
+    sectionStyle: PropTypes.object,
+
+    /** Deprecated property for controlling the style of the parent component */
```

Comment:
Was in charge of controlling the style of the parent component, or was it strictly the width? I remember it as width only, so I wanted to check. Even though it's deprecated, being clear about what it does could help the org that's using an older version using it

Reply:
It looks like it was meant to only do width at some point but switched over to a CSS in JS object. This change is probably for the best since it provides more control. I updated the proptypes to be a bit more explicit with what is going on.